src/ts/components/transaction/Fee.tsx

```diff
@@ -40,14 +62,26 @@ class Fee extends React.Component<IFeeProps, IFeeState> {
   }
 
   private doUpdate = () => {
-    this.api.query.balances.transferFee(currentFee => {
-      console.log('currentFee', currentFee)
-      const formattedFee = formatBalance(currentFee)
+    const address = this.props.address
+    const toAddress = this.props.toAddress
+    Promise.all([
```

Comment:
why not keep `this.api.rpc.system.properties()` here as well?

Reply:
rpc does not support looking up fees.